client/noclip.lua

```diff
@@ -15,12 +15,12 @@ local MinY, MaxY        = -89.0, 89.0
 ]]
 
 -- Perspective values
-local PedFirstPersonNoClip      = false      -- No Clip in first person when not in a vehicle
+local PedFirstPersonNoClip      = true      -- No Clip in first person when not in a vehicle
 local VehFirstPersonNoClip      = false      -- No Clip in first person when in a vehicle
 
 -- Speed settings
 local Speed                     = 1         -- Default: 1
-local MaxSpeed                  = 16.0      -- Default: 16.0
+local MaxSpeed                  = 20.0      -- Default: 16.0
```

Comment:
Unsure if these need to be changed due to them being configurable values for the end-user. Obviously down to the user's preference.

Reply:
just my base speed i like